lib/bundler/dsl.rb

```diff
@@ -216,9 +217,9 @@ def group(*args, &blk)
       args.each { @groups.pop }
     end
 
-    def install_if(*args, &blk)
```

Comment:
Should this have a proper error message then if the user doesn't supply a block?